lessons/meta/installing-naucse/index.md

```diff
@@ -75,7 +63,7 @@ Druhý režim nejdříve vykreslí všechny stránky a až poté ti je zobrazí
 Pustí se následovně (pozor, nějakou chvíli to trvá):
 
 ```console
-(__venv__)$ python -m naucse freeze --serve
+$ pipenv run naucse freeze --serve
```

Comment:
...a tady je.

Nezkoušel jsem, co je správně a jestlit o funguje, ale minimálně mě to zarazilo.